Star Seeker/Views/Presenters/Game.swift

```diff
@@ -141,13 +168,13 @@ extension Game {
 
 /* MARK: -- Extension which gives Game the ability to attend to itself. */
 extension Game {
-    
     /** Sets the required one-time configurations to the passed SKView */
     func setup ( _ view: SKView ) {
         view.allowsTransparency = true
         self.view!.isMultipleTouchEnabled = true
         self.backgroundColor = .clear
         self.physicsWorld.speed = 1
+        self.physicsWorld.gravity = GameConfig.playerGravity
```

Comment:
Why would one aligned the physicsWorld's gravity to be the same as player's?

Reply:
because we have not set any gravity, thus i think defining this value is required. I don't think you can apply gravity directly to player's node unless you apply it using applyForce in sprite kit. since the only node that is affected by gravity is only the player node, i set the world's gravity